src/vmm/src/builder.rs

```diff
@@ -833,7 +840,13 @@ pub fn configure_system_for_boot(
                     vmm.guest_memory(),
                     entry_addr,
                     &vcpu_config,
-                    vmm.vm.supported_cpuid().clone(),
+                    X86_64CpuConfiguration {
+                        cpuid: Cpuid::try_from(
+                            RawCpuid::from(vmm.vm.supported_cpuid().clone()).clone(),
```

Comment:
nit: unnecessary clone()?
```suggestion
                            RawCpuid::from(vmm.vm.supported_cpuid().clone()),
```

Reply:
@zulinx86 You are right. I will fix in [PR/3511](https://github.com/firecracker-microvm/firecracker/pull/3511/).